packages/grafana-runtime/src/utils/plugin.ts

```diff
@@ -3,13 +3,29 @@ import { config } from '../config';
 // @ts-ignore
 import System from 'systemjs/dist/system.js';
 
+/**
+ * Option to specify a plugin css that should be applied for the dark
+ * and the light theme.
+ *
+ * @public
+ */
 export interface PluginCssOptions {
   light: string;
   dark: string;
 }
 
+/**
+ * @internal
+ */
 export const SystemJS = System;
```

Comment:
note: feels a bit dangerous to expose this one via the grafana/runtime

Reply:
yeah, we probably could have some higher level API for plugin loader to be able to interact with SystemJS

Reply:
Can you create issue around this/